supporting-blog-content/fetch-surrounding-chunks/fetch-surrounding-chunks.ipynb

```diff
@@ -39,13 +39,13 @@
   },
```

Comment:
_Commented on notebook `supporting-blog-content/fetch-surrounding-chunks/fetch-surrounding-chunks.ipynb` Cell 5 Line 32_
```python


# Create the client instance
esclient = Elasticsearch(
    cloud_id=ELASTIC_CLOUD_ID,
    api_key=ELASTIC_API_KEY,
)
print(esclient.info())
```
contains UserWarning. suggest not commit this.

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/269)._

Reply:
@joemcelroy  thank you.  Issue resolved and pushed. review requested.